### PR TITLE
Adding an option for HTTPS.

### PR DESCRIPTION
**Pull request by iclanton**

### CODE REVIEW

**Comment by iclanton** [comment]
@nickpape-msft

File: src/ServeTask.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ export interface IServeTaskConfig {
   };
```
Comment (nickpape-msft) [inline src/ServeTask.ts]: add comments for these please

File: src/ServeTask.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ export interface IServeTaskConfig {
...
   initialPage: string;
   port: number;
+  https: boolean;
```
Comment (nickpape-msft) [inline src/ServeTask.ts]: can you add a comment that gives a link to https://nodejs.org/api/https.html#https_https_createserver_options_requestlistener

File: src/ServeTask.ts [hunk 3]
```diff
@@ -49,7 +51,8 @@ export class ServeTask extends GulpTask<IServeTaskConfig> {
       livereload: true,
       middleware: (): Function[] => [logRequestsMiddleware, enableCorsMiddleware],
       port: port,
-      root: rootPath
+      root: rootPath,
+      https: this.taskConfig.https
```
Comment (nickpape-msft) [inline src/ServeTask.ts]: does the API server also need Https?

Comment (iclanton) [inline src/ServeTask.ts]: Good point. Probably, yes.

**Comment by nickpape-msft** [comment]
Approved

**Pull request merged**